As of at least version 3.9.16, Maven refuses to build the project due to
it specifying Java 7, which Maven considers too old.  Bump up the Java
version slightly to Java 8 to please Maven.

--- parent/pom.xml.orig	2016-05-16 01:17:25
+++ parent/pom.xml	2026-08-18 21:08:12
@@ -73,8 +73,8 @@
                <artifactId>maven-compiler-plugin</artifactId>
                <version>3.5.1</version>
                <configuration>
-                  <source>1.7</source>
-                  <target>1.7</target>
+                  <source>1.8</source>
+                  <target>1.8</target>
                </configuration>
             </plugin>
             <plugin>
